### DISCUSSION TITLE
Codenames for versions

### DISCUSSION THREAD

**Discussion by imdevarsh** [CatalystDevOrg/Catalyst#87] [Proposals]
So, for example, `22.01` is a release. Why don't we add something like a codename of sorts?
The name Catalyst already has a kind of scientific meaning, so I thought something like this:
`22.01 - Hydrogen`
`22.02 - Lithium`
`22.03 - Nitrogen`
Some elements. Just a thought.

**Comment by jdev082** [comment]
@webdev03 why?

**Reply by imdevarsh** [reply]
Because it looks cool..? And awesome.

**Reply by imdevarsh** [reply]
bump

**Reply by jdev082** [reply]
@webdev03 each release be named after a cat breed?

**Reply by imdevarsh** [reply]
You mean freecat ???

**Reply by jdev082** [reply]
@webdev03 cat-alyst

**Reply by imdevarsh** [reply]
I thought it had more of a science-y name.

**Reply by jdev082** [reply]
1.0, Scottish Fold?
2.0 Tabby? (darn it should of used this for the first release with tabs, oh wait that's 1.0

1.0 Tabby
2.0 Scottish Fold

**Comment by jdev082** [comment]
1.0 Tabby
2.0 Scottish Fold
3.0 Sand Cat
4.0 Maine Coon
5.0 American Shorthair

Each point release will be code named derivatives of that cat breed codename the main release has been given.

**Comment by jdev082** [comment]
If a code name is too long it'll be for example
American Shorthair

AS_dev-prerelease 1, for ending strings.
